pkg/cmd/step/step_tag_test.go

```diff
@@ -1,12 +1,11 @@
-package step_test
+package step
```

Comment:
Why is this change from a `step_test` package into `step`? I am asking this because the test will get compiled into the `step` package, ending finally into the binary. Do you need any unexported functions from `step` package in tests?

Reply:
> I am asking this because the test will get compiled into the step package, ending finally into the binary

Are you sure about this? I would say it is not included.

Reply:
Same for me, I'm quite new to golang, but I've always seen tests in the same package, unless there is a circular import. In my case yes, I wanted to test a private function. 

https://golang.org/pkg/testing/

> To write a new test suite, create a file whose name ends _test.go that contains the TestXxx functions as described here. Put the file in the same package as the one being tested. The file will be excluded from regular package builds but will be included when the “go test” command is run. For more detail, run “go help test” and “go help testflag”.

Reply:
See the docs https://tip.golang.org/cmd/go/#hdr-Test_packages

Reply:
>Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary.

Reply:
Well there is an easy way to settle this:
- type garbage in `pkg/cmd/step/step_tag_test.go`
- `make build` -> compiles without any problem
- `make build-all` -> shows an error
- type the same garbage in `pkg/cmd/step/step_tag.go`
- `make build` -> fails

`go` does not even bother reading the `*_test.go` files when building, so it means they are not included in the final binary.